client/src/app/site/login/components/login-mask/login-mask.component.ts

```diff
@@ -19,6 +20,8 @@ import { ParentErrorStateMatcher } from 'app/shared/parent-error-state-matcher';
 import { BaseComponent } from 'app/site/base/components/base.component';
 import { BrowserSupportService } from '../../services/browser-support.service';
 
+const HttpWarning = _('Using OpenSlides over a HTTP connection is not supported. Enable HTTPS to continue.');
```

Comment:
This is a sophistry, but should it not be called a_n_ HTTP connection?

Reply:
depends on whether you say aitch or haitch.

> An `aitch tee tee pee` connection
> A `haitch tee tee pee` connection

Perhaps I should avoid using a or an ans simply write "Using OpenSlides over HTTP is not supported"